admin/resources/conservation-and-planning-admin/conservation-and-planning-admin.resource.ts

```diff
@@ -1,9 +1,11 @@
 import { ResourceWithOptions } from "adminjs";
 import { ConservationPlanningAndAdmin } from "@shared/entities/cost-inputs/conservation-and-planning-admin.entity.js";
+import { GLOBAL_COMMON_PROPERTIES, COMMON_RESOURCE_LIST_PROPERTIES } from "../common/common.resources.js";
 
 export const ConservationAndPlanningAdminResource: ResourceWithOptions = {
   resource: ConservationPlanningAndAdmin,
   options: {
+    properties: {...COMMON_RESOURCE_LIST_PROPERTIES, ...GLOBAL_COMMON_PROPERTIES},
```

Comment:
In this case this entity does not have a activity column, but it renders in the UI

Reply:
Ah, I see, I was trying something out and it remained that way, Removing COMMON_RESOURCE_LIST_PROPERTIES  from the properties. 👍